static-site/src/pages/ncov-sit-reps.jsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import Helmet from "react-helmet";
+import Collapsible from "react-collapsible";
+import _sortBy from "lodash/sortBy";
+import {FaFile} from "react-icons/fa";
+import config from "../../data/SiteConfig";
+import NavBar from '../components/nav-bar';
+import MainLayout from "../components/layout";
+import { SmallSpacer, MediumSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import Footer from "../components/Footer";
+import CollapseTitle from "../components/Misc/collapse-title";
+import {parseNcovSitRepInfo} from "../../../auspice-client/customisations/languageSelector";
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {hasError: false};
+    this.handleError = this.handleServerError.bind(this);
+    this.sitRepCards = this.getNarrativesByLanguage.bind(this);
+  }
+
+  getNarrativesByLanguage(json) {
+    const narrativesByLanguage = {};
+    json.narratives
+      .filter((o) => o.request)
+      .map((o) => o.request)
+      .map(parseNcovSitRepInfo)
+      .filter((sitrep) => sitrep !== null)
+      .forEach((sitrep) => {
+        sitrep.url = "/"+sitrep.url;
+        if (narrativesByLanguage[sitrep.languageCode]) {
+          narrativesByLanguage[sitrep.languageCode].narratives.push(sitrep);
+        } else {
+          narrativesByLanguage[sitrep.languageCode] = {
+            languageCode: sitrep.languageCode,
+            languageNative: sitrep.languageNative,
+            narratives: [sitrep]
+          };
+        }
+      });
+    const languageObjects = Object.values(narrativesByLanguage);
+    // sort most recent dates first within each language
+    languageObjects.forEach((l) => {
+      l.narratives.sort().reverse();
+    });
+    // English first then by language code
+    return _sortBy(languageObjects, [(o) => o.languageNative !== "English", (o) => o.languageCode]);
+  }
+
+  handleServerError(response) {
+    if (!response.ok) {
+      throw new Error(`Failed to fetch available situation report narratives from /charon/getAvailable: ${response.status} ${response.statusText}`);
+    }
+    return response;
+  }
+
+  componentDidMount() {
+    fetch(`/charon/getAvailable`)
+      .then(this.handleServerError)
+      .then((res) => res.json())
+      .then((json) => {
+        this.setState({ narrativesByLanguage: this.getNarrativesByLanguage(json) });
+      })
+      .catch(error => {
+        console.log("The following error occured during fetching or parsing situation reports:", error);
+        this.setState({ hasError: true });
+      });
+  }
+
+  render() {
+    return (
+      <MainLayout>
+        <div className="index-container">
+          <Helmet title={config.siteTitle} />
+          <main>
+
+            <NavBar location={this.props.location} />
```

Comment:
The nav-bar component needs to be wrapped as follows such that it renders appropriatly for logged-in users

```js
import UserDataWrapper from "../layouts/userDataWrapper";
...
<UserDataWrapper>
      <NavBar location={this.props.location} />
</UserDataWrapper>
```

P.S. @eharkins ping me on slack & we can make you a username for nextstrain.org